DependencyInjection/Configuration.php

```diff
@@ -70,8 +70,8 @@ private function addResourcesSection(ArrayNodeDefinition $node)
                     ->prototype('array')
                         ->children()
                             ->scalarNode('subject')->isRequired()->cannotBeEmpty()->end()
-                            ->scalarNode('attribute')->isRequired()->defaultValue(Attribute::class)->cannotBeEmpty()->end()
-                            ->scalarNode('option')->isRequired()->defaultValue(Option::class)->cannotBeEmpty()->end()
+                            ->scalarNode('attribute')->isRequired()->defaultValue(ProductAttribute::class)->cannotBeEmpty()->end()
+                            ->scalarNode('option')->isRequired()->defaultValue(ProductOption::class)->cannotBeEmpty()->end()
```

Comment:
Ideally the bundle would not be dependent on `sylius/product`.

I might consider forking these other packages, and adding some sort of compatibility layer with the new version of Sylius.

I don't know how easy this is to achieve however - I might consider merging Variation into Archetype the same as Sylius did for Product. Otherwise this change is okay.